src/callbacks_stage/subcell_bounds_check.jl

```diff
@@ -42,23 +42,35 @@ function (callback::BoundsCheckCallback)(u_ode, integrator, stage)
                   (stage == length(alg.c)) &&
                   (iter % callback.interval == 0 || integrator.finalstep)
     @trixi_timeit timer() "check_bounds" check_bounds(u, mesh, equations, solver, cache,
-                                                      solver.volume_integral, t,
-                                                      iter + 1,
-                                                      callback.output_directory,
-                                                      save_errors)
+                                                      solver.volume_integral)
+
+    if save_errors
+        @trixi_timeit timer() "save_errors" save_bounds_check_errors(callback.output_directory,
+                                                                     u, t, iter + 1,
+                                                                     equations,
+                                                                     solver.volume_integral)
+    end
+end
+
+@inline function check_bounds(u, mesh, equations, solver, cache,
+                              volume_integral::AbstractVolumeIntegral)
+    return nothing
+end
```

Comment:
Probably it makes more sense to delete this routine or, alternatively, include an error message here, such that we get an error when calling `BoundsCheckCallback` for simulations without subcell limiting.

Reply:
The bounds check callback does not make sense for other volume integrals, right? In that case I'd just remove the dummy functions